src/status_im/protocol/web3/delivery.cljs

```diff
@@ -89,7 +89,7 @@
          (fn [messages]
            (when messages
              (let [message (messages id)
-                   message' (dissoc message to)]
+                   message' (dissoc message to nil)]
```

Comment:
This is full of magic at the moment: it is possible that the message with the same id will be sent to different users, that's why when `ack` comes we have to remove pending message by both `message-id` and `from`. But there are scenarios when the message is not sent to some particular user, like group message (at least it works so now), then the recipient is `nil`... Imagine, we send group message and it is added to pending messages map with `[message-id nil]` key, so when `ack` is received we are trying to remove `[message-id from]` and in result the message is not removed from pending messages collection.